psycop/common/model_training_v2/pipeline.py

```diff
@@ -23,4 +23,4 @@ def train_baseline_model(cfg: BaselineSchema) -> float:
     result = cfg.training_method.train()
     result.eval_dataset.to_disk(path=cfg.experiment_path)
 
-    return result.metric
+    return result.metrics
```

Comment:
TrainingResult doesn't have a .metrics attribute? How come this does not result in a warning?

Reply:
it did but was somehow missed